package/MDAnalysis/analysis/rms.py

```diff
@@ -516,24 +558,25 @@ def run(self, start=0, stop=-1, step=1, progout=10, quiet=False):
         This method implements an algorithm for computing sums of squares while
         avoiding overflows and underflows; please reference:
```

Comment:
The sentence should reference the paper with a proper [reST citation](http://docutils.sourceforge.net/docs/user/rst/quickref.html#citations):
```rest
avoiding overflows and underflows; please reference [Welford1962]_.